jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -13,6 +13,11 @@ export IMAGE_OS
 export DEFAULT_HOSTS_MEMORY
 export NUM_NODES
 
+# Overrides docker.io
+export OVERRIDE_DOCKER_IO_REGISTRY=${OVERRIDE_DOCKER_IO_REGISTRY:-"registry.nordix.org/docker-hub-proxy/library"}
+# Overrides registry.hub.docker.com
+export OVERRIDE_DOCKER_HUB_REGISTRY=${OVERRIDE_DOCKER_HUB_REGISTRY:-"registry.nordix.org/docker-hub-proxy"}
```

Comment:
Who is the consumer of this variable?

Reply:
`OVERRIDE_DOCKER_HUB_REGISTRY`

Reply:
other repos:
one example is https://github.com/metal3-io/cluster-api-provider-metal3/pull/155/files